### ISSUE TITLE
🏁 I Don't Care - Notepad ↔ v23.4.2

### ISSUE THREAD

**Issue by ynx0** [itchio/itch-compatibility-watchlist#1253]
:rotating_light: tool [I Don't Care - Notepad](https://rombus.itch.io/i-dont-care) is broken for me.

:book: Here's the complete [debug log](https://gist.github.com/5674cd5ed518a9e0dba572105c3d07e9).

:running: Any additional details can go here!
This line might be the key
`[2017-11-06 @ 20:55:48.159] [launch/native] err: WARNING: Could not open/create prefs root node Software\JavaSoft\Prefs at root 0x80000002. Windows RegCreateKeyEx(...) returned error code 5.`

**Comment by fasterthanlime** [comment]
Yeah, it looks like it's trying to create a registry key but Windows won't let it - and that crashes the app. Maybe it was designed with Windows 7 in mind, which might be more lax about registry permissions?

At any rate, that's a bug with that software, not with the itch.io app, so I recommend contacting the developer directly - you could link that page in their comments section for example!

**Comment by ynx0** [comment]
Oh sorry. I will do that on the weekend probably.

Thank you for taking a look.
